JSDemos/Demos/DataGrid/StateManagement/Angular/app/app.service.ts

```diff
@@ -33,7 +34,12 @@ export class Service {
     }
 
     getOrders(): Observable<Order[]> {
-        this.http.get(`${this.url}/Orders?skip=700`, { withCredentials: true }).subscribe((data) => {
+        if(this.ordersSubscription) {
+            this.ordersSubscription.unsubscribe();
+            this.ordersSubscription = null;
+        }
+
+        this.ordersSubscription = this.http.get(`${this.url}/Orders?skip=700`, { withCredentials: true }).subscribe((data) => {
```

Comment:
it is not necessary to unsubscribe from http get observable.
See https://stackoverflow.com/questions/35042929/is-it-necessary-to-unsubscribe-from-observables-created-by-http-methods